src/sst/core/cfgoutput/jsonConfigOutput.cc

```diff
@@ -139,11 +143,25 @@ JSONConfigGraphOutput::generate(const Config* cfg, ConfigGraph* graph)
 {
     if ( nullptr == outputFile ) { throw ConfigGraphOutputException("Output file is not open for writing"); }
 
-    auto compMap = graph->getComponentMap();
-    auto linkMap = graph->getLinkMap();
+    const auto& compMap = graph->getComponentMap();
+    const auto& linkMap = graph->getLinkMap();
 
     json::ordered_json outputJson;
 
+    // Put in the program options
+    outputJson["program_options"]["verbose"]     = std::to_string(cfg->verbose);
+    outputJson["program_options"]["stopAtCycle"] = cfg->stopAtCycle;
+    if ( cfg->print_timing ) outputJson["program_options"]["print-timing-info"] = "";
```

Comment:
Yes, but since putting that in, I discovered there's a way to use true and false there and have it work correctly in the reader.  I'll change it to the new method.

Reply:
This actually turns out to be an issue with the Config class.  Not sure that I'll be able to resolve it easily.

Reply:
For now, I'm just going to set this field to true in this case.  It will get ignored with the current JSON reader, and just seeing the option in the list will set it to true.  I'll have to fix the Config object to make setting it to false work.